src/mixed-line-bar-chart/__tests__/mixed-chart.test.tsx

```diff
@@ -577,7 +577,7 @@ describe('Axes', () => {
           height={250}
           xDomain={[0, 12]}
           yDomain={[0, 100]}
-          i18nStrings={{ xTickFormatter: (value: number) => value.toFixed(2) }}
+          xTickFormatter={(value: number) => value.toFixed(2)}
```

Comment:
We should probably keep testing the existing props since they are still supported.